compass/ocean/tests/global_ocean/mesh/mesh.py

```diff
@@ -24,7 +25,9 @@ class MeshStep(Step):
         The name of the mesh config file
     """
     def __init__(self, test_case, mesh_name, with_ice_shelf_cavities,
-                 package, mesh_config_filename, name='mesh', subdir=None):
+                 package, mesh_config_filename, name='mesh', subdir=None,
+                 do_inject_bathymetry=False, preserve_floodplain=False,
+                 floodplain_elevation=10.0):
```

Comment:
Could I put this under the existing `[global_ocean]` section? Or would `[shperical_mesh]` make more sense?

Reply:
@xylar, what default value would you recommend using here?

Reply:
@sbrus89, `floodplane_elevation` shouldn't be a parameter here at all.  It should come only from a config option.  In the config file, the default value should still be `10.0`.  I'm okay with using `[global_ocean]` as the config section.  I think it would be better to pull out anything that is used across test groups and is currently in the `global_ocean` test group into the ocean framework but that obviously won't happen in this PR.  So we could change where that config option lives as part of any such future reorganization.

Reply:
I see, I'll change that.